lib/node.js

```diff
@@ -118,6 +118,7 @@ declare class Buffer extends Uint8Array {
 
 declare module "buffer" {
   declare var kMaxLength: number;
+  declare export var Buffer: typeof Buffer;
```

Comment:
i deleted this from https://github.com/facebook/flow/pull/3495 for the same reason